external/CapnProto.cmake

```diff
@@ -114,6 +122,7 @@ foreach (INCLUDE_DIR ${CAPNP_INCLUDE_DIRS})
   install(DIRECTORY ${INCLUDE_DIR}/capnp
```

Comment:
@scottpurdy: I looked at this once more and it was definitely worth a second look. When I run `make install` on my MacBookPro, there are two attempts to install the capnp files under the following headings:

1. `Installing: /Users/vkruglikov/nta/nupic.core/build/release/include//capnp`
2. `Installing: /Users/vkruglikov/nta/nupic.core/build/release/include/capnp`

Note that the headings only differ in two slashes versus a single slash before the last component. This corresponds precisely to the two duplicated sets of install commands that only vary by the presence/absence of an extra slash:

From CapnProto.cmake:
```
foreach (INCLUDE_DIR ${CAPNP_INCLUDE_DIRS})
  install(DIRECTORY ${INCLUDE_DIR}/kj
          DESTINATION include/)
  install(DIRECTORY ${INCLUDE_DIR}/capnp
          DESTINATION include/)
endforeach ()

install(FILES ${CAPNP_LINK_LIBRARIES}
        DESTINATION lib/)
```

From src/CMakeFiles.txt:

```
install(FILES ${CAPNP_LINK_LIBRARIES}
        DESTINATION lib)

foreach(directory ${CAPNP_INCLUDE_DIRS})
  install(DIRECTORY "${directory}/capnp"
          DESTINATION include)
  install(DIRECTORY "${directory}/kj"
          DESTINATION include)
endforeach()

```